dkist/dataset/tiled_dataset.py

```diff
@@ -167,7 +170,26 @@ def _get_axislabels(ax):
                 ylabel = coord.get_axislabel() or coord._get_default_axislabel()
         return (xlabel, ylabel)
 
-    def plot(self, slice_index, share_zscale=False, fig=None, **kwargs):
+    @staticmethod
+    def _ensure_wcs_ordered_axis_lims(wcs: HighLevelWCSWrapper, ax: Axes) -> None:
+        """
+        Adjust axis limits so the WCS values go from smaller numbers in the bottom left to higher numbers in the upper right.
+        """
+        xmin, xmax = ax.get_xlim()
+        ymin, ymax = ax.get_ylim()
+
+        world_xmin, world_ymin = wcs.low_level_wcs.pixel_to_world_values(xmin, ymin)
+        world_xmax, world_ymax = wcs.low_level_wcs.pixel_to_world_values(xmax, ymax)
+
+        new_xmin, new_ymin = wcs.low_level_wcs.world_to_pixel_values(min(world_xmin, world_xmax), min(world_ymin, world_ymax))
+        new_xmax, new_ymax = wcs.low_level_wcs.world_to_pixel_values(max(world_xmin, world_xmax), max(world_ymin, world_ymax))
+
+        ax.set_xlim(new_xmin, new_xmax)
+        ax.set_ylim(new_ymin, new_ymax)
+
+        return
+
+    def plot(self, slice_index, share_zscale=False, fig=None, limits_from_wcs=True, **kwargs):
```

Comment:
I would like to throw a warning here if the user loads an old ASDF file with this new `.plot` to go and tell them to redownload a new ADSF.

Reply:
I'm fine with this idea, but I'm not sure how to implement it. A user will get wrong plots (and should thus be warned) iff these two conditions are BOTH true:

1. `dkist-processing-vbi` version is < 1.19.0
2. `dkist-inventory` version is < 1.?.?

We can check 1 easily, but I don't know about checking the `dkist-inventory` version. Ideas?